src/neo/SmartContract/ApplicationEngine.Runtime.cs

```diff
@@ -77,6 +77,12 @@ partial class ApplicationEngine
         /// </summary>
         public static readonly InteropDescriptor System_Runtime_GetInvocationCounter = Register("System.Runtime.GetInvocationCounter", nameof(GetInvocationCounter), 1 << 4, CallFlags.None);
 
+        /// <summary>
+        /// The <see cref="InteropDescriptor"/> of System.Runtime.GetRandom.
+        /// Gets the random number generated from the VRF.
+        /// </summary>
+        public static readonly InteropDescriptor System_Runtime_GetRandom = Register("System.Runtime.GetRandom", nameof(GetRandom), 1 << 4, CallFlags.None);
```

Comment:
The price needs to be recalculated. @neo-project/ngd-shanghai

Reply:
After testing result shows that this syscall costs about 3 times of NOP: a vm with 2000000 NOP costs about 1.5 s while another with same count of _GetRandom_ calls costs about 4.5 s

Reply:
> After testing result shows that this syscall costs about 3 times of NOP: a vm with 2000000 NOP costs about 1.5 s while another with same count of _GetRandom_ calls costs about 4.5 s

@Qiao-Jin What should the price be~

Reply:
I think this price is OK

Reply:
@Qiao-Jin Can you test it again? The algorithm is changed.

Reply:
After testing result shows that this syscall costs about 3 times of NOP: a vm with 2000000 NOP costs about 1.5 s while another with same count of GetRandom calls costs about 4.3 s, approximately the same with former result